File: src/main/java/eu/mjindra/utils/dices/D12.java
package eu.mjindra.utils.dices;

/**
 * A D12 die.
 *
 * @author Martin Jindra
 * @version 31.05.2023
 */
public class D12 extends Dice {

    public D12(byte amount) {
        this(amount, (byte) 0);
    }

    public D12(byte amount, byte modifier) {
        super(amount, (byte) 12, modifier);
    }

}

File: lib/dnd/dices/d12.dart
import 'package:helios/dnd/dices/dice.dart';
import 'package:helios/dnd/dices/die.dart' show Die;

/// A D12 die.
class D12 extends Die {
  D12() : super(DiceSides.d12);

  D12.mod(int modifier) : super.mod(DiceSides.d12, modifier);
}
